deploy-idsvr-aws-eks.sh

```diff
@@ -27,14 +27,13 @@ greeting_message() {
   echo "| [6] NGINX PHANTOM TOKEN PLUGIN                                             |"
   echo "| [7] EXAMPLE NODEJS API                                                     |"
   echo "|----------------------------------------------------------------------------|" 
-  echo -e "\n"
 }
 
 
 pre_requisites_check() {
   # Check if aws cli, kubectl, eksctl, helm & jq are installed
-  if ! [[ $(aws --version) && $(helm version) && $(jq --version) && $(eksctl version) ]]; then
-      echo "Please install aws cli, kubectl, eksctl, helm & jq to continue with the deployment .."
+  if ! [[ $(aws --version) && $(helm version) && $(jq --version) && $(eksctl version) && $(terraform version) ]]; then
+      echo "Please install aws cli, kubectl, eksctl, terraform, helm & jq to continue with the deployment .."
```

Comment:
should say `eksctl or terraform` as the user can choose which one to use.

Reply:
Good catch ! , Fixed